dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -407,6 +408,96 @@ def new_integration
     render template: Policies::Lti.early_access? ? 'lti/v1/integrations/early_access' : 'lti/v1/integrations'
   end
 
+  # GET /lti/v1/dynamic_registration
+  # Initiate LTI Dynamic Registration flow
+  def new_registration
```

Comment:
I recommend creating a separate `Lti::V1::DynamicRegistrationController` to avoid overloading the already bloated `LtiV1Controller`

Reply:
Yeah, that's probably a good idea. I remember now that there are some examples where you've done that already. Will take a look 👍

Reply:
You can also check this: https://github.com/code-dot-org/code-dot-org/pull/58464/files#diff-a3ae4dbdd699d7712c2672e40f767aa0a7ff3c5a36ad8adb624d676be890f444

Reply:
✅ This work has been done.